extension/include/boost/di/extension/injections/factory.hpp

```diff
@@ -9,6 +9,7 @@
 #include <memory>
 
 #include "boost/di.hpp"
+#include "extensible_injector.hpp"
```

Comment:
I think that should be the whole path "boost/extension/extensible_injector.hpp", to be more explicit and also for online compiler to work

Reply:
It will make user to put all extension by full path. I wasn't going to do it as I have specific project structure and it will not be the part of boost::di library